### PR TITLE
move python test entrypoint to makefile

### PR DESCRIPTION
**Pull request by chebbyChefNEQ**
move python test entrypoint to makefile so we can manage pytest marks easily.

### CODE REVIEW

File: python/Makefile
Comment (wjones127) [inline python/Makefile:1]: Oh boy I'm gonna add soooo many things to this 😀

File: python/Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+ifeq ($(CI), true)
+    PYTEST_ARGS = -x -v --durations=30
```
Comment (wjones127) [inline python/Makefile:2]: Are we sure we want early exit? I kind of like to know all the failures, especially if it's a weird platform.

Comment (wjones127) [reply python/Makefile:2]: Durations is cool though 👍

Comment (chebbyChefNEQ) [reply python/Makefile:2]: Just copied these over. I'm down to remove the -x

Comment (chebbyChefNEQ) [reply python/Makefile]: updated

**Review by wjones127 — APPROVED** [review]

**Pull request merged**